.circleci/config.yml

```
@@ -231,18 +231,29 @@ commands:
       - npm_install
       - *attach-workspace
       - run:
-          environment:
-            # pkg doesn't use a normal cache directory by default
-            # see https://github.com/vercel/pkg/issues/1225
-            PKG_CACHE_PATH: tmp/pkg/
-          name: Run dist command with the appropriate argument
+          # Need this to make sure pkg cache is scoped to specific pkg version
+          name: Get pkg version
+          command: jq '.devDependencies.pkg' cli/package.json > .pkgversion
+      - restore_cache:
+          keys:
+            - pkg-v1-{{ checksum ".pkgversion" }}
+      - run:
+          name: Run dist script
           command: yarn run dist --version "<<parameters.version>>"
       - persist_to_workspace:
           root: ./
           paths:
             - dist/
             # Needed for the alpine docker build
             - tmp/pkg/
```

does this need to be changed as well, or is this a separate thing?
```suggestion
            - tmp/pkg-fetch/
```